ansible/configs/kni-osp/files/clouds.yaml.j2

```diff
@@ -0,0 +1,12 @@
+clouds:
+  {{ osp_project_name }}:
+    auth:
+      auth_url: "{{ osp_auth_url }}"
+      username: "{{ guid }}-user"
+      project_name: "{{ osp_project_name }}"
+      project_id: "{{ osp_project_id }}"
+      user_domain_name: "Default"
+      password: "{{ heat_user_password }}"
```

Comment:
This should be changed to use the new var available - `osp_auth_password_member`

Reply:
done in last commit